assignments/this.js

```diff
@@ -1,26 +1,93 @@
 /* The for principles of "this";
-* in your own words. explain the four principle for the "this" keyword below.
-*
-* 1. 
-* 2. 
-* 3. 
-* 4. 
-*
-* write out a code example of each explanation above
-*/
+ * in your own words. explain the four principle for the "this" keyword below.
+ *
+ * 1. Window/Global Binding
+ * 2. Implicit Binding
+ * 3. New binding
+ * 4. Explicit binding
+ *
+ * write out a code example of each explanation above
+ */
 
 // Principle 1
-
+// In the global scope 'this' applies to window or global space
 // code example for Window Binding
 
-// Principle 2
+function sayName(name) {
+  //console.log(this);
```

Comment:
```suggestion
  console.log(this);
```
uncomment this line, it's where the global this principle is being demonstrated